int16 FIR_filter_coeffs[120] 
---
int8 confirm